.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,18 @@
+# when a github release happens, publish an npm package,
+on:
+  release:
+    types: [released]
+  # support manual release
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: tag that needs to publish
+        type: string
+        required: true
+jobs:
+  npm:
+    uses: salesforcecli/github-workflows/.github/workflows/npmPublish.yml@main
```

Comment:
Is there a reason we are using node version `lts/*` [here](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/npmPublish.yml#L27). I've seen `latest` used everywhere else so far.

Reply:
[This comment](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/npmPublish.yml#L55-L56) can be removed

Reply:
Description is incorrect on `ctcClose` ([source](https://github.com/salesforcecli/github-workflows/blob/main/.github/actions/ctcClose/action.yml#L2))

Reply:
> Is there a reason we are using node version lts/* [here](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/npmPublish.yml#L27). I've seen latest used everywhere else so far.

I didn't think of a good reason to change it from the current behavior.  Not anything clever or important.